### DISCUSSION TITLE
How to become a member of Julia language

### DISCUSSION THREAD

**Discussion by izam-mohammed** [JuliaLang/julia#48315] [General]
I am really curious about the process of become a member of the Julia language since my pr is merged. Can someone explain that??? Please

**Comment by vtjnash** [comment]
The main part of the process is getting several PRs merged, so you become recognized as a contributor. Typically after a few contributions what happens is that someone nominates a new contributor and the existing contributors vote whether to send an invitation. Frequently if someone expresses interest, we will give out 'triage' permission pretty freely, which enables someone to manage PRs, including applying the 'merge me' tag.

So if you want to become a member, the briefest advice I can give is to be around, making and commenting on PRs. Being active and present on Slack and discourse is also helpful.

**Reply by izam-mohammed** [reply]
I will do that 💓
